rose/web/index.html

```diff
@@ -14,6 +14,8 @@
         <button id="start" disabled>Start</button>
         <button id="stop"  disabled>Stop</button>
 
+        <button id="toggle_mute"><img src="res/icons/mute.svg"></button>
```

Comment:
This possibly starts with the wrong icon, I wonder if we should have a third icon for the loading step, or make this button disabled until we read the state from local storage.